boards/pba-d-01-kw2x/Makefile.dep

```diff
@@ -0,0 +1,4 @@
+ifneq (,$(filter ng_netif,$(USEMODULE)))
+  USEMODULE += kw2xrf
```

Comment:
>don't we do the dependencies the other way around? see #2947.

It's not even merged.

>ifneq (,$(filter kw2xrf,$(USEMODULE)))
  USEMODULE+=ng_ieee802154

Where to put it?

Please write in clear sentences, not in riddles.